packages/core/src/codewhisperer/explorer/codewhispererChildrenNodes.ts

```diff
@@ -61,6 +62,41 @@ export function createAutoSuggestions(type: 'item' | 'tree', pause: boolean): an
     }
 }
 
+export function createAutoScans(type: 'item', pause: boolean): DataQuickPickItem<'autoScans'>
+export function createAutoScans(type: 'tree', pause: boolean): TreeNode<Command>
+export function createAutoScans(
+    type: 'item' | 'tree',
+    pause: boolean
+): DataQuickPickItem<'autoScans'> | TreeNode<Command>
+export function createAutoScans(type: 'item' | 'tree', pause: boolean): any {
+    const labelResume = localize('AWS.codewhisperer.resumeCodeWhispererNode.label', 'Resume Auto-Scans')
+    const iconResume = getIcon('vscode-debug-alt')
+    const labelPause = localize('AWS.codewhisperer.pauseCodeWhispererNode.label', 'Pause Auto-Scans')
+    const iconPause = getIcon('vscode-debug-pause')
+
+    switch (type) {
+        case 'tree':
+            return toggleCodeScans.build(placeholder, cwTreeNodeSource).asTreeNode(
+                pause
+                    ? {
+                          label: labelPause,
+                          iconPath: iconPause,
+                      }
+                    : {
+                          label: labelResume,
+                          iconPath: iconResume,
+                      }
+            )
+        case 'item':
+            return {
+                data: 'autoScans',
+                label: pause ? codicon`${iconPause} ${labelPause}` : codicon`${iconResume} ${labelResume}`,
+                description: pause ? 'Currently RUNNING' : 'Currently PAUSED',
```

Comment:
"Currently" is never needed in a UI status. The status is assumed to be "current". If it wasn't current then typically it would say "most recent" or "last".
```suggestion
                description: pause ? 'RUNNING' : 'PAUSED',
```